src/streamlink/plugins/chzzk.py

```diff
@@ -305,7 +306,7 @@ def _get_vod_playback(self, datatype, data):
             ).items():
                 if stream.video_representation and stream.video_representation.mimeType == "video/mp2t":
                     yield name, stream
-            return
+                    return
```

Comment:
This one does not belong here, hence why I didn't comment on that. See what's iterated on here, namely the mapping of DASH streams. A return statement here would result in only the first stream quality being returned, which is not what we want.

Reply:
I just re-examined my code and understood what you told me.
It was quite a foolish mistake...